### PR TITLE
Updating About activity layout. Adding new README file and Gradle test dependencies

### PR DESCRIPTION
**Pull request by ghost**
I'd updated About activity layout and added also test dependencies for Robolectric and Espresso.

### CODE REVIEW

**Comment by ghost** [comment]
I've added new README.md file. Hope you'll like it.

File: app/src/main/res/layout/about_fragment.xml [hunk 1]
```diff
@@ -1,12 +1,125 @@
...
+                android:id="@+id/aboutDesc"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_centerHorizontal="true"
+                android:layout_marginBottom="0dp"
+                android:layout_marginTop="5dp"
+                android:fontFamily="sans-serif-light"
+                android:gravity="center_horizontal"
+                android:text="@string/description"
+                android:textAppearance="?android:attr/textAppearanceMedium" />
```
Comment (pjakubczyk) [inline app/src/main/res/layout/about_fragment.xml:79]: extract to style -> eventDate

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <resources>
-    <string name="app_name">Droidcon Poland</string>
-
+    <string name="app_name">Droidcon Krakow 2015</string>
```
Comment (pjakubczyk) [inline app/src/main/res/values/strings.xml]: either Cracow, either Kraków

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -22,4 +22,9 @@
     <string name="lunch_break">Lunch break</string>
     <string name="session">Session</string>
     <string name="barcamp">Barcamp</string>
+    <string name="date">4th-6th December 2015</string>
+    <string name="more_info">More informations:  </string>
+    <string name="website_link">droidcon.pl/</string>
+    <string name="aboutDesc">Want to hear about latest Android technology and talk to experienced Android expert speakers? Be a part of it! Join second edition of droidcon Kraków!</string>
```
Comment (pjakubczyk) [inline app/src/main/res/values/strings.xml]: change to name to something I can understand

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -22,4 +22,9 @@
...
+    <string name="description">Want to hear about latest Android technology and talk to experienced Android expert speakers? Be a part of it! Join second edition of droidcon Kraków!</string>
```
Comment (pjakubczyk) [inline app/src/main/res/values/strings.xml]: what description ?

File: app/build.gradle [hunk 5]
```diff
@@ -58,4 +58,16 @@ dependencies {
     compile 'io.realm:realm-android:0.82.2'
 
     compile 'com.makeramen:roundedimageview:2.2.0'
+
+    testCompile 'junit:junit:4.12'
```
Comment (pjakubczyk) [inline app/build.gradle:62]: I see more dependencies which are not used :>

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (ghost) [inline app/src/main/res/values/strings.xml]: I doubled the same text. Well, the second one deleted.

File: app/build.gradle (hunk 5 above)
Comment (ghost) [inline app/build.gradle:62]: well, Espresso UI and Robolectric tests will come soon. Now I added only some extremely simple examples ;-)

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (ghost) [inline app/src/main/res/values/strings.xml]: changed

**Pull request merged**